### DISCUSSION TITLE
Pi 5 with NVME - small Frigate setup gamechanger?

### DISCUSSION THREAD

**Discussion by FrigateEnthusiast** [blakeblackshear/frigate#12056] [General]
This recent article on Hackster caught my eye:

https://www.hackster.io/news/benchmarking-tensorflow-and-tensorflow-lite-on-raspberry-pi-5-b9156d58a6a2

A Pi 5 running Tensorflow Lite Mobile Net v1/v2 was on par with a Coral Edge TPU.  And now with NVME add on cards being out and about, this could be a gamechanger for small Frigate setups.

Just tossing this out for food for thought...

**Comment by hawkeye217** [comment]
There has been a recent discussion about this here: https://github.com/blakeblackshear/frigate/discussions/2050

This board would be a good candidate for one of the [Community Supported Boards](https://docs.frigate.video/development/contributing-boards/). The maintainers don't have any bandwidth (nor do they have the hardware) to support this themselves at this time.

See https://github.com/blakeblackshear/frigate/issues/11663#issuecomment-2141808697

**Reply by FrigateEnthusiast** [reply]
Saw a comment about no h264 hw decoder, but if all your cams were outputting h264 streams, that might not be an issue.

What stood out in this article were the native (no Hailo-8 hat) inference times.

**Reply by hawkeye217** [reply]
For sure, it looks pretty quick.
